pkg/controller/deployment/deployment_controller.go

```diff
@@ -536,7 +548,8 @@ func (dc *DeploymentController) syncRecreateDeployment(deployment *extensions.De
 	}
 	if scaledDown {
 		// Update DeploymentStatus
-		return dc.updateDeploymentStatus(allRSs, newRS, deployment)
+		cond := deploymentutil.NewCondition(extensions.DeploymentProgressing, api.ConditionTrue, "ScaledDown", "Deployment has been scaled down")
```

Comment:
I'm not sure ScaledUp/ScaledDown conditions make sense. When would these conditions be true?